core/src/models/DestinationGroupMembership.ts

```diff
@@ -40,6 +40,8 @@ export class DestinationGroupMembership extends CommonModel<DestinationGroupMemb
   @BelongsTo(() => Destination)
   destination: Destination;
 
+  uniqueIdentifier = ["groupId", "destinationId", "remoteKey"];
```

Comment:
Previously `destinationId` + `groupId` was one unique pair, and `destinationId` + `remoteKey` another.
Unsure if this needs to be expanded into a `[["destinationId", "groupId"], ["destinationId", "remoteKey"]]`.